sentry/resource_sentry_project.go

```diff
@@ -143,14 +153,20 @@ func resourceSentryProjectCreate(ctx context.Context, d *schema.ResourceData, me
 
 	org := d.Get("organization").(string)
 	team := d.Get("team").(string)
+	var teams []interface{}
+	if team == "" {
+		teams = d.Get("teams").(*schema.Set).List()
+		team = teams[0].(string)
+	}
 	params := &sentry.CreateProjectParams{
 		Name: d.Get("name").(string),
 		Slug: d.Get("slug").(string),
 	}
 
 	tflog.Debug(ctx, "Creating Sentry project", map[string]interface{}{
-		"teamName": team,
-		"org":      org,
+		"team":  team,
+		"teams": teams,
+		"org":   org,
 	})
 	proj, _, err := client.Projects.Create(ctx, org, team, params)
```

Comment:
`teams` is a set. It makes `teams[0]` random, and this function call will have random values.
If no other way around to pass teams, maybe you should sort `teams` first and then get the first one?

Reply:
This `set` [implementation](https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/set.go#L97-L98) has deterministic ordering, so, given the same values, the order of the output will always be the same. 

I've added an in-code comment since this is confusing: f7ba4afef304056b2ebdc98c14a9c84c5c66aa85

Reply:
Perfect :-D Thanks Taj,